build.xml

```diff
@@ -696,7 +696,15 @@
         <scm connection="${scm.connection}" developerConnection="${scm.developerConnection}" url="${scm.url}"/>
   <dependency groupId="com.google.guava" artifactId="guava"/>
       </artifact:pom>
-
+      <artifact:pom id="test-pom"
+                    artifactId="cassandra-test"
+                    url="https://cassandra.apache.org"
+                    name="Apache Cassandra">
+      <parent groupId="org.apache.cassandra"
+              artifactId="cassandra-parent"
+              version="${version}"/>
+      <scm connection="${scm.connection}" developerConnection="${scm.developerConnection}" url="${scm.url}"/>
+      </artifact:pom>
```

Comment:
Do we need any dependencies here?

Reply:
I don't believe so. The dependency works when tested in the sls repo